### PR TITLE
Fix code blocks secret.md

### PR DESCRIPTION
**Pull request by juzna**
The formatting is currently broken because code blocks are not correctly terminated.

### CODE REVIEW

File: content/en/docs/concepts/configuration/secret.md [hunk 1]
```diff
@@ -619,10 +625,8 @@ start until all the pod's volumes are mounted.
 Create a secret containing some ssh keys:
 
 ```shell
-kubectl create secret generic ssh-key-secret --from-file=ssh-privatekey=/path/to/.ssh/id_rsa 
-```
-```
---from-file=ssh-publickey=/path/to/.ssh/id_rsa.pub
+kubectl create secret generic ssh-key-secret --from-file=ssh-privatekey=/path/to/.ssh/id_rsa \
```
Comment (DanyC97) [inline content/en/docs/concepts/configuration/secret.md]: do we need `\` ? we haven't formatted/ done on other commands and so will be good to keep consistency

**Comment by DanyC97** [comment]
@juzna you mind fixing the conflict file? thx

**Comment by mistyhacks** [comment]
Please resolve the conflicts. Thanks!

**Comment by DanyC97** [comment]
@juzna are you still interested  in this PR ?

**Pull request merged**